prog/bootstrap_rhizome.rb

```diff
@@ -10,11 +10,6 @@ def user
   end
 
   label def start
-    sshable.update(raw_private_key_1: SshKey.generate.keypair) if sshable.raw_private_key_1.nil?
```

Comment:
As I know some vm services such as PostgreSQL uses bootstrap rhizome to install packages to the vm. Does this affect these services? cc @byucesoy

Reply:
Yes had a brief chat about this with @byucesoy earlier. I just checked, it seems to have usage in PostgreSQL, MinIO and inference_endpoint as well. Wondering if we should defer this removal to a separate change.

Reply:
Chatted with @byucesoy on this, going to revert this how it was before. We can take this one up in a separate change.